pkg/handler/v1/provider/openstack.go

```diff
@@ -1,12 +1,9 @@
 /*
 Copyright 2020 The Kubermatic Kubernetes Platform contributors.
-
```

Comment:
You should remove those changes in the license header ;-)

Reply:
on it!

Reply:
Done